python-frontend/src/main/java/org/sonar/plugins/python/api/SubscriptionContext.java

```diff
@@ -42,6 +45,10 @@
 
   PythonFile pythonFile();
 
+  Map<String, Set<Symbol>> stubFilesSymbols();
```

Comment:
do we really need to return a Map? AFAICT, ExternalTypeHierarchies in sonar-sec doesn't need to know anything about modules.
I think we can just return `Collection<Symbol>` with the union of all external symbols.
Moreover I think we can drop the 'builtinSymbols' API and merge them in this returned collection